internal/lib/control_config.go

```diff
@@ -252,3 +256,16 @@ func (c *akoControlConfig) PodEventf(eventType, reason, message string, formatAr
 		}
 	}
 }
+
+func (c *akoControlConfig) SetLicenseType() {
+	response, err := utils.LicenseTypeFromURI()
+	if err != nil {
+		utils.AviLog.Warnf("Unable to fetch system configuration, error %s", err.Error())
```

Comment:
@metalcyanide : Should we return from here in case of error? Else we might get error at line number 266